kafka-ui-api/src/test/java/com/provectus/kafka/ui/util/jsonschema/AvroJsonSchemaConverterTest.java

```diff
@@ -14,9 +14,9 @@
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 
-public class AvroJsonSchemaConverterTest {
+class AvroJsonSchemaConverterTest {
   @Test
-  public void avroConvertTest() throws URISyntaxException, JsonProcessingException {
+  void avroConvertTest() throws URISyntaxException, JsonProcessingException {
```

Comment:
As far as I remember, junit tests should be public. If that's the case, let's make sonar ignore this stuff for test classes.

Reply:
regarding java:S5786, JUnit5 test classes and methods should have default package visibility ([docs](https://junit.org/junit5/docs/current/user-guide/#writing-tests-classes-and-methods))